### DISCUSSION TITLE
Test rules locally

### DISCUSSION THREAD

**Discussion by marnheus** [tfsaggregator/aggregator-cli#250] [Q&A] [answered]
Question : I want to test a rule locally, but it looks like I never get a full event with updated fields. Is there a way to emulate that?
Thanks.

**Comment by giuliov** [comment, marked as answer]
No, need to change implementation.
[Line 445 of AggregatorRules](https://github.com/tfsaggregator/aggregator-cli/blob/db747eb5dd6886da46e989d62b7227caa85a8f1d/src/aggregator-cli/Rules/AggregatorRules.cs#L455) retrieves the original work item record instead of synthetizing one.

It should not be hard to change that.
